src/libsyntax/feature_gate.rs

```diff
@@ -609,6 +606,8 @@ declare_features! (
     (accepted, fn_must_use, "1.27.0", Some(43302), None),
     // Allows use of the :lifetime macro fragment specifier
     (accepted, macro_lifetime_matcher, "1.27.0", Some(34303), None),
+    // Used to preserve symbols (see llvm.used)
+    (accepted, used, "1.28.0", Some(40289), None),
 );
 
 // If you change this, please modify src/doc/unstable-book as well. You must
```

Comment:
Do we even have an entry in the `unstable-book`? If yes, apply this comment, if no, we should probably document it together with stabilization.

Reply:
> Do we even have an entry in the unstable-book?

Yes.

> If yes, apply this comment

I asked in the PR description where the documentation should be moved to as there are a few options (reference, book, RBE, etc.)

Reply:
I would have assumed to add it to the `book`, because that would have been where I'd have searched for it.

Reply:
@steveklabnik thinks otherwise (cf. https://github.com/rust-lang/rust/pull/51366#issuecomment-394690836 and https://github.com/rust-lang/rust/pull/51366#issuecomment-394779024). I'll prep a PR to add documentation to the reference and the nomicon.